files/cf-exclude-include.json

```diff
@@ -1,5 +1,6 @@
 {
   "globalExcludes": [
+    "zume",

```

Comment:
Sorry to be picky, but can you add it in the alphabetical spot for it?

Reply:
is it okay now? maybe we should add that as a comment to the file